app/javascript/elm/src/Main.elm

```diff
@@ -48,6 +48,7 @@ type alias Model =
     , isCrowdMapOn : Bool
     , crowdMapResolution : Int
     , timeRange : TimeRange
+    , indoor : Bool
```

Comment:
have you considered `isIndoor` here?

Reply:
yes! and now you mention it I'll actually do it :P